src/components/views/right_panel/UserInfo.tsx

```diff
@@ -132,8 +135,16 @@ async function openDMForUser(matrixClient: MatrixClient, userId: string) {
             createRoomOptions.encryption = true;
         }
     }
+    createRoom(createRoomOptions)
+    return modal.close();
```

Comment:
I am not really sure, maybe returning `createRoom(createRoomOptions)` would be better.

Reply:
It's up to you or a member of the dev team